src/LendingPool.sol

```diff
@@ -349,12 +350,12 @@ contract LendingPool is Owned, TrustedProtocol, DebtToken, InterestRateModule {
 
         unchecked {
             //gas: can't overflow for reasonable interest rates
-            base = 1e18 + interestRate;
+            base = 1e18 + interestRatePerYear;
 
-            //gas: only overflows when blocks.number > 894262060268226281981748468
+            //gas: only overflows when blocks.timestamp > 894262060268226281981748468
             //in practice: assumption that delta of blocks < 341640000 (150 years)
```

Comment:
Assumptions here are no longer correct, should be:
            //gas: only overflows when (block.timestamp - lastSyncedBlockTimestamp) > 1e59
            //in practice: exponent in LogExpMath lib is limited to 130e18,
            //Corresponding to a delta of timestamps of 4099680000 (or 130 years),
            //much bigger than any realistic time difference between two syncs.